lib/phoenix/endpoint/adapter.ex

```diff
@@ -60,7 +82,13 @@ defmodule Phoenix.Endpoint.Adapter do
      secret_key_base: nil,
      server: Application.get_env(:phoenix, :serve_endpoints, false),
      url: [host: "localhost"],
-     pubsub: []]
+     pubsub: [],
+
+     # Assets
+     assets: [
+       watchers: [],
```

Comment:
I don't think watchers are specific to assets. It happens that our default watcher is an asset related thing but they are completely distinct. To me it would be another config on root.